app/_kong_plugins/confluent/index.md

```diff
@@ -33,10 +37,26 @@ icon: confluent.png
 categories:
   - transformations
 
+related_resources:
+  - text: Kafka Log
+    url: /plugins/kafka-log/
+  - text: Kafka Upstream
+    url: /plugins/kafka-upstream/
+
 search_aliases:
   - confluent
```

Comment:
```suggestion
  - kafka
```
Since confluent is already the name of the plugin, that search alias doesn't add anything.  But would be good for this plugin to appear in a kafka search.